### ISSUE TITLE
run script problems

### ISSUE THREAD

**Issue by glensc** [dboissier/mongo4idea#210]
not very intuitive to run script.js, i would had expected to open "console" somewhere in mongodb settings, like you have with sql console.

anyway, to execute script.js, i need to use Run (control+shift+r).

few problems with it:
1. it creates new run configuration each time i press it
2. if i configure default db (run->edit configurations->defaults->mongo->database), then it is not saved (tried to set "doctrine" instead of "admin"), and each run configuration is still created with "admin" database.
3. if i finally "run" the script, it does nothing, because the script name is not passed on commandline:
```
/usr/local/bin/mongo localhost:27017/doctrine
MongoDB shell version v3.6.4
connecting to: mongodb://localhost:27017/doctrine
MongoDB server version: 3.4.10-2.10
WARNING: shell and server versions do not match
```

**Comment by mikerudenko** [comment]
@glensc Do you solve this problem?

**Comment by glensc** [comment]
@mikerudenko no. nothing done, no feedback from maintainer, so i just avoided this tool.
